code/game/objects/items/eightball.dm

```diff
@@ -169,7 +171,7 @@
 	var/top_vote
 
 	for(var/vote in votes)
-		var/amount_of_votes = length(votes[vote])
+		var/amount_of_votes = votes[vote]
```

Comment:
`votes` is an assoc list of vote = number of votes, so this `length()` always returned `0`